docs/readme.md

```diff
@@ -7,10 +7,12 @@
 You may wish to launch a local version of our documentation website
 to test changes for formatting, link resolution, etc.
 
-Python 3 is required:
+Python 3 is required. It is also good practice to use virtualenv:
 ```
 cd /path/to/wfl/docs
-python3 -m pip install -r requirements.txt
+virtualenv <<name>>
+source <<name>>/bin/activate
+python3 pip install -r requirements.txt
```

Comment:
Think this should be `python3 -m pip install…`.

Reply:
I just used the above, but can change it no prob :)

Reply:
Thank you!  (FTR, the missing `-m` was a documentation bug that I recently fixed, so perhaps referencing an old version.)